docs/core/install/linux-fedora.md

```diff
@@ -104,7 +92,11 @@ This section provides information on common errors you may get while using the p
 
 ### Unable to find package
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+For more information on installing .NET without a package manager, see one of the following articles:
+
+- [Install the .NET SDK or the .NET Runtime with Snap.](../linux-snap.md)
+- [Install the .NET SDK or the .NET Runtime with a script.](../linux-scripted-manual.md#scripted-install)
+- [Install the .NET SDK or the .NET Runtime manually.](../linux-scripted-manual.md#manual-install)
```

Comment:
These links are wrong since they were taken from `./includes/` folder

```suggestion
- [Install the .NET SDK or the .NET Runtime with Snap.](linux-snap.md)
- [Install the .NET SDK or the .NET Runtime with a script.](linux-scripted-manual.md#scripted-install)
- [Install the .NET SDK or the .NET Runtime manually.](linux-scripted-manual.md#manual-install)
```